app/controllers/angular_template_controller.rb

```diff
@@ -0,0 +1,5 @@
+# used to serve static angular templates from under app/views/angular_template/
```

Comment:
@himdel this is a great PR, thanks for doing it.
I wonder - why the folder is named angular_template and not angular_templates in plural?